apps/common/views/partials/back.html

```diff
@@ -0,0 +1,5 @@
+{{$back}}
+  {{#backLink}}
+    <a href="{{#url}}{{backLink}}{{/url}}">Back</a>
```

Comment:
I have decided against this in the end as there are a lot of text in the partials templates that need to be added to the translations and like you said, this should probably exist some where in hof rather than the form it self. What do you think? 
@JoeChapman @gxxm